.storybook/webpack.config.js

```diff
@@ -142,18 +141,23 @@ module.exports = async ({
     })
   }
 
-  config.module.rules.push({
+  config.module.rules.push(
+    {
```

Comment:
Looks like some prettier diff here- just want to be sure this is ok and not due to an out-of-sync config.